repose-aggregator/src/docs/asciidoc/filters/index.adoc

```diff
@@ -2,15 +2,104 @@
 
 include::../_includes/in-progress.adoc[]
 
-<<add-header.adoc#,Add Header Filter>>::
-    Enables users to add new headers and header values to a request and/or response.
+== <<add-header.adoc#,Add Header Filter>>
```

Comment:
Making these headers makes the font size huge and the page unwieldy. Could we use a smaller header size and still be in the TOC?